specification/search/data-plane/Azure.Search/preview/2023-07-01-Preview/searchindex.json

```diff
@@ -0,0 +1,2724 @@
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "scoringProfile",
+            "in": "query",
+            "type": "string",
+            "description": "The name of a scoring profile to evaluate match scores for matching documents in order to sort the results.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "semanticConfiguration",
+            "in": "query",
+            "type": "string",
+            "description": "The name of the semantic configuration that lists which fields should be used for semantic ranking, captions, highlights, and answers",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "semanticErrorHandling",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "partial",
+              "fail"
+            ],
+            "x-ms-enum": {
+              "name": "SemanticErrorHandling",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "partial",
+                  "name": "Partial",
+                  "description": "If the semantic processing fails, partial results still return. The definition of partial results depends on what semantic step failed and what was the reason for failure."
+                },
+                {
+                  "value": "fail",
+                  "name": "Fail",
+                  "description": "If there is an exception during the semantic processing step, the query will fail and return the appropriate HTTP code depending on the error."
+                }
+              ]
+            },
+            "description": "Allows the user to choose whether a semantic call should fail completely, or to return partial results.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "semanticMaxWaitInMilliseconds",
+            "in": "query",
+            "type": "integer",
+            "format": "int32",
+            "minimum": 700,
+            "description": "Allows the user to set an upper bound on the amount of time it takes for semantic enrichment to finish processing before the request fails.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "debug",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "disabled",
+              "speller",
+              "semantic",
+              "all"
```

Comment:
should speller and all be removed from here?

Reply:
same in the post version as well

Reply:
Good catch, I removed the value descriptions in the other PR but I guess I forgot to remove them here.